src/tasks.js

```diff
@@ -0,0 +1,30 @@
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const addTask = (description, completed, index) => {
+  const task = new Task(description, completed, index);
+  const oldList = JSON.parse(localStorage.getItem('list'));
+  oldList.push(task);
+  localStorage.setItem('list', JSON.stringify(oldList));
+  return task;
+};
+
+const removeTask = (index) => {
+  const oldList = JSON.parse(localStorage.getItem('list'));
+  const newList = oldList.filter((ind) => ind.index !== index);
+  if (newList.length > 0) {
+    let i = 1;
+    newList.forEach((element) => {
+      element.index = i;
+      i += 1;
+    });
+  }
+  localStorage.setItem('list', JSON.stringify(newList));
+};
+
```

Comment:
- [ ] In the project's specific requirements, a function for editing task descriptions is required to be implemented. You can add it here or any other file that suits you. Kindly add the edit description method. :pray: You can revisit the project's requirements [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md).
- [ ] In the project's specific requirements, the function for the "Clear all completed" button (use filter() method) is required to be implemented. Again you can revisit the project's requirements [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md).